addons/website_sale/static/src/js/website_sale_tour_shop_zoom.js

```diff
@@ -35,7 +36,8 @@ tour.register('shop_zoom', {
     },
     {
         content: "wait for variant to be loaded",
-        trigger: '.oe_currency_value:contains("21.00")'
+        trigger: '.oe_currency_value:contains("21.00")',
+        run: function () {},
```

Comment:
Are you sure this fixes something? We should be able to click on not interactive items during the tour and have the same tour behavior :/

Reply:
This one does nothing indeed, but the one on the cart button is the one fixing the issue.

I revert those that are not useful then.